src/main/java/com/example/event_manager/controller/EventController.java

```diff
@@ -18,21 +24,30 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.ModelAndView;
 
-@RequestMapping("events")
 @Controller
 @AllArgsConstructor()
+@RequestMapping("events")
 public class EventController {
 
   private final PersonService personService;
   private final EventService eventService;
   private final TaskStatusService taskStatusService;
 
+  @GetMapping(value = "billingsRaport", produces = "application/pdf")
+  public ResponseEntity<byte[]> Billings(@RequestParam final Long id)
```

Comment:
minor issue but java convention for method names is to start with lowercase character, so please rename it to "billings"